packages/eas-cli/src/credentials/manager/ManageIosBeta.ts

```diff
@@ -0,0 +1,125 @@
+import { CommonIosAppCredentialsFragment, IosDistributionType } from '../../graphql/generated';
+import { getProjectAccountName } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+import { findAccountByName } from '../../user/Account';
+import { ensureActorHasUsername } from '../../user/actions';
+import { Action, CredentialsManager } from '../CredentialsManager';
+import { Context } from '../context';
+//import { RemoveDistributionCertificateBeta } from '../ios/actions/RemoveDistributionCertificateBeta';
+import { AppLookupParams } from '../ios/api/GraphqlClient';
+import {
+  displayAllIosCredentials,
+  displayEmptyIosCredentials,
+} from '../ios/utils/printCredentialsBeta';
+import { PressAnyKeyToContinue } from './HelperActions';
+
+enum ActionType {
+  SetupBuildCredentials,
+  SetupBuildCredentialsFromCredentialsJson,
+  UpdateCredentialsJson,
+  UseExistingDistributionCertificate,
+  RemoveSpecificProvisioningProfile,
+  CreateDistributionCertificate,
+  UpdateDistributionCertificate,
+  RemoveDistributionCertificate,
+}
+
+export class ManageIosBeta implements Action {
+  private _currentDistributionType = null as IosDistributionType | null;
+
+  chooseDistributionType(
```

Comment:
If we want to keep this behavior this function should be part of the displayAllIosCredentials implementation. (an argument for changing that behavior in comments below)

Reply:
yeah i ended up getting rid of this